### ISSUE TITLE
firefox doesn't work after nvda restart

### ISSUE THREAD

**Issue by beqabeqa473** [nvaccess/nvda#7011]
hi all.

1. start firefox and navigate to some url.
2. restart nvda.

expected results, browsemode should continue to work.
actual results, browsemode is not working .

the solution is to close and open firefox.

**Comment by josephsl** [comment]
Hi, isn’t this a duplicate of several issues regarding Gecko engine or similar? Thanks.

**Comment by Brian1Gaff** [comment]
This has been an intermittent issue for some time, indeed on one machine it 
actually made firefox vanish completely.
 It can often also affect any program using a virtual buffer, liike email 
software content displays and often Adobe reader, but its rare.
 No idea what goes on unless its the hooks that are removed or something.
 The fact that its not always doing it seems to support that some software 
is more picky as to when you do such things than they used to be.

I don't recall this issue in firefox before version 48 myself.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by jcsteh** [comment]
Duplicate of #5758. Only happens if Firefox is focused when you restart. The workaround is to alt+tab out of Firefox before restarting NVDA.
